core/perf_test/CMakeLists.txt

```diff
@@ -64,6 +64,13 @@ IF(Kokkos_ENABLE_OPENMPTARGET)
   )
 ENDIF()
 
+IF((KOKKOS_ENABLE_CUDA AND KOKKOS_ENABLE_CUDA_LAMBDA) OR KOKKOS_ENABLE_HIP OR KOKKOS_ENABLE_SYCL)
```

Comment:
It seems like the dependence on `KOKKOS_ENABLE_CUDA_LAMBDA` could be pretty trivially eliminated, since it's just one bit of code that would need to be converted to a named functor

Reply:
good catch, I already did this but forgot to remove the guard

Reply:
done in [8bf552](https://github.com/kokkos/kokkos/pull/5289/commits/8bf5526aa5baf9b6d2dcb59acf41f2736109a0e6)